src/wallet/wallet.h

```diff
@@ -497,20 +497,23 @@ class COutput
     /** Whether we know how to spend this output, ignoring the lack of keys */
     bool fSolvable;
 
+    /** Whether this output is a watch-only output */
+    bool use_max_sig;
```

Comment:
In "Use 72 byte dummy signatures when watching only inputs may be used ":

I know this flag is used for watch-only outputs, but that's not what the flag is, it's a flag that uses the 72 byte dummy signature instead of the 71 byte one. Noting that this is enabled when output is watch-only is useful, of course.